examples/custom-server-express/server.js

```diff
@@ -0,0 +1,37 @@
+const express = require('express')
+const next = require('next')
+const accepts = require('accepts')
+
+const app = next('.', { dev: true })
```

Comment:
I believe that explicit is better than implicit; please, don't hide this feature in the example.